1.0/tools/tests.md

```diff
@@ -0,0 +1,385 @@
+    var ajax;
+    var request;
+    var server;
+    var responseHeaders = {
+      json: { 'Content-Type': 'application/json' }
+    };
+    setup(function() {
+      server = sinon.fakeServer.create();
+      server.respondWith(
+        'GET',
+        /\/responds_to_get_with_json.*/, [
+          200,
+          responseHeaders.json,
+          '{"success":true}'
+        ]
+      );
+    });
+    teardown(function() {
+      server.restore();
+    });
+    suite('when making simple GET requests for JSON', function() {
+      setup(function() {
+        // get fresh instance of iron-ajax before every test
+        ajax = fixture('simple-get');
+      });
+      test('has sane defaults that love you', function() {
+        request = ajax.generateRequest();
+        server.respond();
+        expect(request.response).to.be.ok;
+        expect(request.response).to.be.an('object');
+        expect(request.response.success).to.be.equal(true);
+      });
+      test('has the correct xhr method', function() {
+        request = ajax.generateRequest();
+        expect(request.xhr.method).to.be.equal('GET');
+      });
+    });
+  });
+</script>
+{% endhighlight %}
+
+The example above uses Chai's [`expect`](http://chaijs.com/api/bdd/) assertion
+style. 
+{: .alert .alert-info }
+
+## Run a subset of tests {#test-subsets}
+
+To run a subset of tests, create an HTML file and call `loadSuites()`. When
+running `wct`, specify the path to the HTML file as the first argument 
+(for example, `wct test/my-test-subset.html`.
+
+{% highlight html %}
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src=”../bower_components/webcomponentsjs/web-components-lite.js”></script>
+    <script src=”../bower_components/web-component-tester/browser.js”></script>
+  </head>
+  <body>
+    <script>
+      WCT.loadSuites([
+        'basic.html',
+        'async.html'
+      ]);
+    </script>
+  </body>
+</html>
+{% endhighlight %}
+
+The argument to `loadSuites()` should be an array of strings. Each string
+should be a relative URL to a test suite. You can configure your tests 
+using query strings in the URLs. See [Test Shadow DOM](#shadow-dom)
+for an example.
+
+## Asynchronous tests
+
+To create an asynchronous test, pass `done` as an argument to the test function
+and then call `done()` when the test is complete. The `done` argument is a
+signal to Mocha that the test is asynchronous. When Mocha runs the tests it 
+waits until it encounters the `done()` call, and eventually times out if it 
+does not encounter it. 
+
+    test('fires lasers', function(done) {
+      myEl.addEventListener('seed-element-lasers', function(event) {
+        assert.equal(event.detail.sound, 'Pew pew!');
+        done();
+      });
+      myEl.fireLasers();
+    });
+
+## Test Shadow DOM {#shadow-dom}
+
+To test out how a test suite behaves when the browser runs native
+Shadow DOM, create a [test subset](#test-subsets) and pass `dom=shadow` as 
```

Comment:
Here (& below), lc "shadow" unless it's the first word of a title or sentence, or you're giving the name of the specification. This is the style we've adopted for the standards (so: shadow DOM, HTML imports, custom elements, not Shadow DOM, HTML Imports, Custom Elements).